dkc/core/models/folder.py

```diff
@@ -62,16 +62,17 @@ class Meta:
     # # Prevent deletion of quotas while a folder references them
     # quota = models.ForeignKey(Quota, on_delete=models.PROTECT)
 
+    @property
     def path_to_root(self) -> List[Folder]:
-        folder = self
-        path = [folder]
-        while folder.parent is not None:
-            folder = folder.parent
-            path.append(folder)
-            if len(path) > self.MAX_TREE_HEIGHT:
-                raise MaxFolderDepthExceeded()
-
-        return path[::-1]
+        return Folder.objects.raw(
+            'WITH RECURSIVE ancestors AS ('
+            'SELECT * FROM core_folder WHERE id=%s'
+            ' UNION ALL'
+            ' SELECT f.* FROM core_folder f INNER JOIN ancestors a ON f.id=a.parent_id'
+            ')'
+            ' SELECT * FROM ancestors',
+            [self.pk],
+        )[::-1]
```

Comment:
This is easy to fix, but I think native order of iteration (root last) actually makes more sense for most usages.

Reply:
Good point, especially given the name of the method.